microshift_configuring/microshift-using-config-tools.adoc

```diff
@@ -0,0 +1,16 @@
+:_content-type: ASSEMBLY
+[id="microshift-using-config-tools"]
+= MicroShift Version
```

Comment:
Elsewhere someone suggested using the attribute in the title to get the product name, so we may want that. At the very least this looks like the title from the version section in troubleshooting.

```suggestion
= Configuring MicroShift
```